.github/workflows/main.yml

```diff
@@ -26,3 +26,14 @@ jobs:
         with:
           browser: chrome
           headless: true
+
+      - name: Report
+        run: npm run combine:reports
+        
+      - name: Report generation 
+        run: npm run generate:reports
```

Comment:
is this required? given that we are generating the reports in the pre and post test script?